### ISSUE TITLE
problem about the result

### ISSUE THREAD

**Issue by SimonLliu** [bnulihaixia/Deep_metric#29]
Hello, thanks for your good jobs!
When I use your codes, i find almost all types of loss can reach the result that rank1 is over 0.60. But when i read the paper, if the method is over 0.60(rank1), it can be seen as a good method. Even the contrastive loss can achieve rank1 over 0.60. Why? Thanks

**Comment by bnulihaixia** [comment]
Network: Inception V2  
Freeze BN 
Dimension: 512 
Adam 1e-5 learning rate 
And utilizing all the pairs (different from the lifted structure randomly sampling) 

All these together is enough to make good performance with contrasstive loss.

**Comment by SimonLliu** [comment]
Thanks very much !

**Comment by SimonLliu** [comment]
> Network: Inception V2
> Freeze BN
> Dimension: 512
> Adam 1e-5 learning rate
> And utilizing all the pairs (different from the lifted structure randomly sampling)
> 
> All these together is enough to make good performance with contrasstive loss.

Wow, i used the contrastive loss in Cub200 and  then got rank1 0.68(but highest baseline in your chart is 0.66). When you train the model , have you use the data augmentation 'rand_crop' which is in your codes?
Thanks for your reply!

**Comment by bnulihaixia** [comment]
yes

**Comment by IAAI-CVResearchGroup** [comment]
Found in my experiment, why BNInception in your code If there is no freeze BN-layer's result will be about two points lower than after freeze.  Why is there no such phenomenon in the InceptionBN network in other people's papers? 

Looking forward to your reply.
